tools/phpcs-excludelist.json

```diff
@@ -538,6 +538,7 @@
 	"projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-posts.php",
 	"projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-queue.php",
 	"projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-sender.php",
+	"projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-search.php",
```

Comment:
This doesn't seem to be preventing the linter from failing for some reason.

Reply:
I'm afraid not. You would still need to have code that is free of linting errors for all new and modified lines in the file (so the whole file in this case).